xml/art_jeos_quickstart.xml

```diff
@@ -216,6 +216,35 @@
     </para>
      </listitem>
     </varlistentry>
+    <varlistentry>
+     <term>systemd-coredump disabled</term>
+     <listitem>
+      <para>
+       Because of the small amount of memory and disk space available by default on JeOS images,
+       systemd-coredump is disabled by default on JeOS images.
+       If your usage scenario requires to collect kernel core dumps for analyzing application
+       crashes:
+       <itemizedlist>
+        <listitem>
+         <para>
+	  Install the <package>systemd-coredump</packages> package,
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+         Set <link xlink:href="https://www.freedesktop.org/software/systemd/man/systemd-coredump.html">kernel.core_pattern</link>
+         in <link xlink:href="https://www.suse.com/support/kb/doc/?id=000018634">sysctl</link>
+         </para>
+        </listitem>
+        <listitem>
```

Comment:
Do you really want the order to be:
- Install package
- Configure it
- Read the documentation

It's closer to reality, yes, but we don't want it to be ;-)

Reply:
Well it's the how to use/analyse documentation when there is a crash.